src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
I think this library should remain agnostic as to the user's reasons for showing Notifications, be it by polling, web sockets, push, long running job, etc. Feature detecting the Push API reveals a bias towards the Push API, but it does not fit most reasons why you would want to notify a user about something. I think that feature detecting the Push API should be outside the scope of Notify.js